src/content/home/helxFeatures.mdx

```diff
@@ -0,0 +1,57 @@
+import GridItem from "../../components/sections/features";
+import { GridContainer } from "../../components/grid.js";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper"
+import applications from "../../images/home/applications-workflow-home.svg";
+import search from "../../images/home/complex-search-func-icon-home.svg";
+import openSource from "../../images/home/open-source-icon-home.svg";
+import scalable from "../../images/home/scalable-icon-home.svg";
+import flexible from "../../images/home/flexibility-icon-home.svg";
+import metadata from "../../images/home/metadata-driven-icon-home.svg";
+
+<MaxWidthWrapper backgroundColor="#F5F6F5" maxWidth="x1" gutterBottom>
+
+<GridContainer spacing={6} padding="2rem 4rem 2rem 8rem">
```

Comment:
i actually dont think you need padding here, but if you are gonna have padding, this one makes the container off center because there is 4rem of padding on the right and 8 rem of padding on the left (/* top | right | bottom | left */) [mdn docs](https://developer.mozilla.org/en-US/docs/Web/CSS/padding)